ccd-definition/AuthorisationComplexType/AuthorisationComplexType.json

```diff
@@ -1,26 +0,0 @@
-[
```

Comment:
Same comment, that file could be fix as separate PR, will be easier to revert just in case.